vite.config.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/namespace
```

Comment:
Thank you for this one. But it doesn't really add functionality to the project. It is not a mandatory change here.